src/Altinn.Notifications/Validators/EmailNotificationOrderRequestValidator.cs

```diff
@@ -23,8 +23,10 @@ public EmailNotificationOrderRequestValidator()
             .WithMessage("A valid email address must be provided for all recipients.");
 
         RuleFor(order => order.RequestedSendTime)
-          .Must(sendTime => sendTime >= DateTime.UtcNow.AddMinutes(-5))
-          .WithMessage("Send time must be in the future. Leave blank to send immediately.");
+            .Must(sendTime => sendTime.Kind != DateTimeKind.Unspecified)
+            .WithMessage("No time zone specified.")
```

Comment:
We could be slightly more specific in the error message. Suggestion:
"The requested send time value must have specified a time zone."